cmd/pls.go

```diff
@@ -69,3 +96,55 @@ func runExporter() {
 		log.Fatal("Error writing to the output file", "file", outputFile, err)
 	}
 }
+
+func runList() {
+	if outputFormat != "plain" && outputFormat != "json" {
+		log.Fatal("Invalid output format. Must be one of plain, json", "format", outputFormat)
+	}
+
+	sqlDB := db.Db()
+	ds := persistence.New(sqlDB)
+	ctx := auth.WithAdminUser(context.Background(), ds)
+
+	options := model.QueryOptions{Sort: "owner_name"}
+
+	if userID != "" {
+		user, err := ds.User(ctx).FindByUsername(userID)
+
+		if err != nil && !errors.Is(err, model.ErrNotFound) {
+			log.Fatal("Error retrieving user by name", "name", userID, err)
+		}
+
+		if errors.Is(err, model.ErrNotFound) {
+			user, err = ds.User(ctx).Get(userID)
+			if err != nil {
+				log.Fatal("Error retrieving user by id", "id", userID, err)
+			}
+		}
+
+		options.Filters = squirrel.Eq{"owner_id": user.ID}
+	}
+
+	playlists, err := ds.Playlist(ctx).GetAll(options)
+	if err != nil {
+		log.Fatal(ctx, "Failed to retrieve playlists", err)
+	}
+
+	if outputFormat == "plain" {
+		println("playlist name|playlist id|owner name|owner id")
+		for _, playlist := range playlists {
+			fmt.Printf("%s|%s|%s|%s\n", playlist.Name, playlist.ID, playlist.OwnerName, playlist.OwnerID)
```

Comment:
IMO, ID should come first. Also I think we should add more fields, at least `Public`

Reply:
Feel free to add other fields as needed. The core is there though (although if more fields, csv writer might be better)

Reply:
Yeah, I think CSV would be better than this _plain_ format

Reply:
Unless we make it more (human) readable. Than we could have the three formats